collections/README.md

```diff
@@ -72,12 +77,12 @@ elsewhere. If this is done, it is recommended to include a `rel` of `canonical`
 
 ## Endpoints
 
-| Endpoint                      | Returns        | Description                                                                                                            |
-| ----------------------------- | -------------- | ---------------------------------------------------------------------------------------------------------------------- |
-| `/`                           | Catalog        | Landing Page and root Catalog                                                                                          |
-| `/api`                        | OAFeat OpenAPI | The OpenAPI service description. The path for this endpoint is only recommended to be `/api`, but may be another path. |
-| `/collections`                | JSON           | Object with a list of Collections contained in the catalog and links                                                   |
-| `/collections/{collectionId}` | Collection     | Returns single Collection JSON                                                                                         |
+| Endpoint                      | Returns                   | Description                                                                                                            |
+| ----------------------------- | ------------------------- | ---------------------------------------------------------------------------------------------------------------------- |
+| `/`                           | Catalog                   | Landing Page and root Catalog                                                                                          |
+| `/api`                        | any | The service description. The path for this endpoint is only recommended to be `/api`, but may be another path. |
```

Comment:
The formatting is weird. The PR adds spaces to all columns although unnecessary, but the line with only relevant change doesn't have any additional whitespace around?

Reply:
will fix. I think the auto-formatter just made an odd decision here.

Reply:
fixed.